src/plugins/admonition/schema.js

```diff
@@ -36,7 +36,7 @@ export default {
         admonition: {
             data: {
                 type: v => TYPES.includes(v),
-                class: c => c === undefined || typeof c === 'string',
+                class: c => c == null || Array.isArray(c),
```

Comment:
As I said, I would prefer it to be an `Immutable.List`. And since we keep classes in a list/array, we should also check that there are no spaces in classes, 'c.every(x => x.match(/\s/) == null)` should work both for `List` and for `Array`.

Reply:
I've changed it to Immutable List.
I'm already handling spaces in class names in front and i'm not sure about putting this into schema validation because it will remove all classes even if only one of them is wrong.